cpp/benchmarks/fixture/benchmark_fixture.hpp

```diff
@@ -68,6 +70,12 @@ inline auto make_pool()
  */
 class benchmark : public ::benchmark::Fixture {
  public:
+  benchmark() : ::benchmark::Fixture()
+  {
+    const char* env_iterations = std::getenv("CUDF_BENCHMARK_ITERATIONS");
```

Comment:
I'm not keen on limiting the number of benchmark iterations for all benchmarks arbitrarily like this. GBench/NVbench run as many iterations as is needed to determine statistical significance.

Reply:
This is not for actual benchmarking results. This is just to enable benchmark binaries run as part of CI or nightly builds to ensure there is nothing broken in benchmarks.